src/app.module.ts

```diff
@@ -21,12 +21,8 @@ import { BlockProviderModule } from '@/block-data-providers/block-provider.modul
             imports: [ConfigModule],
             inject: [ConfigService],
             useFactory: (configService: ConfigService) => ({
-                type: 'postgres',
-                host: configService.get<string>('db.host'),
-                port: configService.get<number>('db.port'),
-                username: configService.get<string>('db.username'),
-                password: configService.get<string>('db.password'),
-                database: configService.get<string>('db.databaseName'),
+                type: 'sqlite',
+                path: configService.get<string>('db.path'),
```

Comment:
This needs to be `database`
```suggestion
                database: configService.get<string>('db.path'),
```